src/Service/SecretsManager/src/SecretsManagerClient.php

```diff
@@ -297,6 +303,20 @@ protected function getEndpointMetadata(?string $region): array
                     'signService' => 'secretsmanager',
                     'signVersions' => ['v4'],
                 ];
+            case 'ca-central-1':
```

Comment:
Missing from the change log?

Reply:
Actually the default case is 
```

        return [
            'endpoint' => "https://secretsmanager.$region.amazonaws.com",
            'signRegion' => $region,
            'signService' => 'secretsmanager',
            'signVersions' => ['v4'],
        ];
```

Which is identical.

But I added an entry for `ca-central-1-fips`